### ISSUE TITLE
Allow console/rcon to edit player configs.

### ISSUE THREAD

**Issue by zerevoidd** [thexaero/open-parties-and-claims#456]
Would be very useful for server utility if you allowed console/rcon to edit player configs in general.
I'm pretty sure this used to be a feature but seems to be changed for some reason.
Please add this back.

**Comment by thexaero** [comment]
Is there a reason why you're not editing them using the UI screen?

**Comment by zerevoidd** [comment]
If I'm personally unavailable to be in-game it's just much more convenient to use commands through console if someone would like more claim chunks. It just seems like an unnecessary feature exclusive to this mod.

**Comment by thexaero** [comment]
I see. I wasn't arguing, just curious.

**Comment by zerevoidd** [comment]
Yeah of course I didn't mean to come off that way, would just be very useful for server administrators to use through console.
